gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineCircleCi.java

```diff
@@ -40,6 +41,7 @@ public final class BaselineCircleCi implements Plugin<Project> {
     @Override
     public void apply(Project project) {
         project.getPluginManager().apply(JunitReportsRootPlugin.class);
+        project.getPluginManager().apply(JunitReportsPlugin.class);
```

Comment:
This shouldn't be necessary. In https://github.com/palantir/gradle-baseline/pull/1764 we explicitly created the `JunitReportsRootPlugin` that does this.

https://github.com/palantir/gradle-baseline/blob/6452ab19fcdb7cfa9c85b8a85e6eeec0bc3abd4d/gradle-junit-reports/src/main/java/com/palantir/gradle/junit/JunitReportsRootPlugin.java#L60